CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+* When you think you're ready to submit your contribution, again insure that you have the latest version of the repository, incase something changed while you were working on your edits.
+* Test your code, to make sure nothing is unexpectedly broken.
+* Once the tests past, you should push your changes to your forked GitHub repository. This can be done through the command line, using the command,
+
+```
+git push origin my-topic-branch
+```
+
+* Issue a [pull request](https://help.github.com/articles/using-pull-requests) on the GitHub website to requset that we merge your changes. One of the American Gut developers will review your code. If we request changes (which is highly probable), *do not issue a new pull request*. Your pull request will be updated automatically.
+
+## Coding Guidelines
+
+We adhere to the [PEP 8](http://www.python.org/dev/peps/pep-0008/) python coding guidelines for code and documentation standards. Before submitting any code to scikit-bio, you should read these carefully and apply the guidelines in your code.
+
+## Testing Guidelines
+Code submitted to American Gut should be unit tested as much as possible. Tests should be added to the *test* directory. 
+
+# IPython Notebooks for Analysis
+We use [IPython Notebooks]() to document and describe analyses performed on American Gut data. There are two primary goals of these notebooks. First, they create a way to reproducibly generate results. They also provide an opportunity for scientific communication and public outreach.
+
+## Audience
+IPython Notebooks are intended to be read by a broad audience. For instance, they may be useful to individual participants, interested in a particular topic, or members of the scientific community looking for more information about the analysis being performed in a paper. **We may need to have a separate set of notebooks for papers?**
+
+Notebooks represent a *sample calculation* for analysis. If a similar set of analysis techniques is applied to the same set of data, multiple notebooks do not need to be produced for each analysis. The results of all the collected analyses represented by the notebook will then be hosted at [site].
+<br>--or--<br>
+Each notebook represents a new analysis. This means that if the same analysis pipeline is applied to multiple metadata categories, each unique analysis should be documented in a notebook.
+
+We publish analysis links on the American Gut blog. We recommend providing both documentation for the code used, and logic behind the analysis. A lengthy introduction section, or text describing variables may also be useful.
+
+
+## Index Notebook
+Please update the [index notebook]() when a new analysis is added.
+
+## Sections
+IPython Notebooks should more or less follow the outline of a paper.
+
+### Credits and License
```

Comment:
Just a few minor typos that I caught:

line 17: +<br/>You should the following information in your bug report: Should say: "You should include the following information..."

Line 22: +* For documentation issues, please first post an issue describing what you propose to add, where you’d like to add it, and a description of why it is an important addition/ For: The "/" should be a "."

Line 29: +* Your analyses adheres to: analyses is plural, so they adhere (not adheres) to. Otherwise, "your analysis adheres to."

Line 88: +American Gut analyses are distributed under a BSD license. Please also provide a data of analysis. Typical text is: Not sure what "a data of analysis" is. Do you mean "data used in analysis?"